torch/testing/_internal/common_methods_invocations.py

```diff
@@ -570,10 +566,10 @@ def sample_inputs_broadcast_to(op_info, device, dtype, requires_grad):
         ((), (1, 3, 2)),
     )
 
-    return tuple(SampleInput((make_tensor(size, device, dtype,
-                                          low=None, high=None,
-                                          requires_grad=requires_grad), shape))
-                 for size, shape in test_cases)
+    return (SampleInput((make_tensor(size, device, dtype,
+                        low=None, high=None,
+                        requires_grad=requires_grad), shape))
+            for size, shape in test_cases)
```

Comment:
Generator expressions are used for some tests. Now they are fully supported without converting to a Sequence with `__len__`.